client/components/data/query-site-monitor-settings/index.jsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { isRequestingSiteMonitorSettings } from 'state/selectors';
+import { requestSiteMonitorSettings } from 'state/sites/monitor/actions';
+
+class QuerySiteMonitorSettings extends Component {
+	static propTypes = {
+		siteId: PropTypes.number.isRequired,
+		requestingSiteMonitorSettings: PropTypes.bool,
+		requestSiteMonitorSettings: PropTypes.func
+	};
+
+	componentWillMount() {
```

Comment:
Addressed in https://github.com/Automattic/wp-calypso/pull/13169/commits/9b480e7794545436a111637aa5c2720dd282d9cf.